src/AutoRest.CSharp/LowLevel/Output/LowLevelClient.cs

```diff
@@ -69,6 +70,10 @@ public LowLevelClient(string name, string ns, string description, string library
                 .OrderBy(m => m.LongRunning != null ? 2 : m.PagingInfo != null ? 1 : 0) // Temporary sorting to minimize amount of changed files. Will be removed when new LRO is implemented
                 .ToArray();
 
+            ConvenienceMethods = cadlInput
+                ? ClientMethods.Select(clientMethod => BuildConvenienceMethod(clientMethod, builder)).ToArray()
```

Comment:
Better to directly cast to `List`.
```suggestion
                ? ClientMethods.Select(clientMethod => BuildConvenienceMethod(clientMethod, builder)).ToList()
```